### ISSUE TITLE
Accessibility issue: missing  space before glossary terms that come after (a, i, b, SS)

### ISSUE THREAD

**Issue by johnnyporkchops** [fecgov/fec-cms#4745]
### Summary

The glossary term  Draftail extension seems to be missing  a space-between when placed after other Draftail items (a, italic, bold, or sans-serif). 

This was fixed visually in https://github.com/fecgov/fec-cms/pull/4742, but the source still has the words concatenated without a space-between, so this is a potential accessibility issue for screen readers.

There was also a [review  comment](https://github.com/fecgov/fec-cms/pull/4742#issuecomment-873081776) brought up by @djgarr about the placement of the glossary icon. It was moved 1% to the left in the above PR to accommodate for it being intermittently cutoff on its right edge:

<img width="240" alt="Screen Shot 2021-07-07 at 5 10 16 PM" src="https://user-images.githubusercontent.com/5572856/124829286-70e9b000-df46-11eb-82a7-f0fedae53934.png">

To see the word-concatenation  issue:
-  view the source for https://www.fec.gov/help-candidates-and-committees/registering-political-party/qualifying-political-party/. In the second paragraph you can see that there is no space between the link and the glossary item after: `higher limits on the` --and-- `contributions`

<img width="930" alt="Screen Shot 2021-07-06 at 11 08 56 AM" src="https://user-images.githubusercontent.com/5572856/124624212-b1fc9a00-de4a-11eb-919a-44be0977e0e3.png">

 - Another confirmation of this is to  copy/paste this from the browser, and although you see  a visual space in browser, the pasted content will not have a space.
    - Copy/pasted directly from browser:
 higher limits on thecontributions they raise than other committees.

Completion criteria:
 - [ ] Glossary links that come after other draftail RTE items (a, italic, bold, or sans-serif) have proper spacing both visually and in source.
 - [ ] Avoid extending the underline on an a:link tag beyond the last character(underlining the space after)
